src/components/layer-controller/styles.js

```diff
@@ -22,19 +25,34 @@ export const useOptionStyles = makeStyles(theme => ({
   },
 }));
 
-export const useExpansionPanelStyles = makeStyles(() => ({
+const sharedControllerStyles = {
+  width: '100%',
+  flexDirection: 'column',
+};
+
+export const useControllerSectionStyles = makeStyles(() => ({
   root: {
-    width: '100%',
-    flexDirection: 'column',
+    ...sharedControllerStyles,
+    padding: '0px 8px',
   },
 }));
 
-export const useExpansionPanelSummaryStyles = makeStyles(theme => ({
+export const StyledExpansionPanelDetails = withStyles(() => ({
+  root: {
+    ...sharedControllerStyles,
+    padding: '8px 8px 24px 8px',
+  },
+}))(ExpansionPanelDetails);
+
+export const StyledExpansionPanelSummary = withStyles(theme => ({
   root: {
-    top: theme.spacing(-1),
     textOverflow: 'ellipsis',
     overflow: 'hidden',
     whiteSpace: 'nowrap',
+    padding: '0px 8px',
```

Comment:
@john-conroy Would it be weird to do something like this? 
```suggestion
    padding: '12px 8px 0px 8px',
    '&$expanded': {
      padding: '0px 8px',
    },
```
Noticed:
<img width="403" alt="Screen Shot 2021-06-07 at 12 04 31 PM" src="https://user-images.githubusercontent.com/43999641/121053659-95a50900-c789-11eb-8d31-0f9dd0325e88.png">

Reply:
I don't think we need to, I should just switch `useOptionStyles` to use `withStyles` instead of `makeStyles`. You'll get back the z-index.

Reply:
Removing `overflow: hidden` from the expansion summary root styles fixes the slider tooltip clipping issue. Was there a reason that this was introduced?

Reply:
Yeah, to prevent something like this:
<img width="417" alt="Screen Shot 2021-06-07 at 12 54 40 PM" src="https://user-images.githubusercontent.com/43999641/121059558-98a2f800-c78f-11eb-917d-5b91faa9db26.png">

Reply:
Wouldn't `overflow:hidden` at this point also hide the slider then?

Reply:
Yeah that's sort of why I opened #956.  Sorry for this mess, I am trying to figure out when this happened because the controller didn't always do this.